project/AppDependencies.scala

```diff
@@ -23,7 +23,7 @@ private object AppDependencies {
 
 object MainDependencies {
   private val bootstrapPlay = "5.16.0"
```

Comment:
There is a newer version of bootstrap to update too. 6.4 I believe

Reply:
To be fixed in a later story due to complexity